src/App.php

```diff
@@ -203,7 +212,8 @@ protected function configureIoc( \Slim\Helper\Set $container ) {
 	protected function configureView( View $view ) {
 		$view->replace( [
 			'app' => $this->slim,
-			'i18nCtx' => $this->slim->i18nContext
+			'i18nCtx' => $this->slim->i18nContext,
+			'supportedLanguages' => $this->getSupportedLanguages(),
```

Comment:
Talking of codesniffer, do you fancy reviewing this? https://gerrit.wikimedia.org/r/#/c/324376/ :-)